### DISCUSSION TITLE
2023/06/22/at-first/

### DISCUSSION THREAD

**Discussion by giscus** [clayicarus/blog.clay.moe#10] [Announcements]
# 2023/06/22/at-first/

从两年前开始，一直在心心念念如何去完成这个网站，有种有心栽花花不开，无心插柳柳成荫的美。不管怎么说，这也是一个好的开始，后续的打算是将以前写过的博客都搬运到这边来，同时也将好好完善一下这个网站。

https://clay.moe/2023/06/22/at-first/

**Comment by pienness** [comment]
天荒地老在哪里呢

**Reply by clayicarus** [reply]
要写一辈子博客吗
